get-started-microservices-on-kubernetes/simple-microservice-llm/src/scripts/disable_opentelemetry.cli

```diff
@@ -0,0 +1,4 @@
+if (outcome == success) of /extension=org.wildfly.extension.opentelemetry:read-resource
```

Comment:
This shouldn't be needed. I have it on my pp because I was doing explicit OTEL trancing